apps/backoffice-v2/src/pages/Entity/components/EditableDetails/EditableDetails.tsx

```diff
@@ -38,12 +38,6 @@ export const EditableDetails: FunctionComponent<IEditableDetails> = ({
   workflowId,
 }) => {
   const [formData, setFormData] = useState(data);
-  const useInitialCategorySetValue = () => {
```

Comment:
There were two instances of the same useEffect. Now the one outside of the `EditableDetails` component is in use. The logic is identical and it just looks like a bad conflict resolution.